solidity/ecdsa/test/utils/inactivity.ts

```diff
@@ -16,7 +16,7 @@ export async function signOperatorInactivityClaim(
 }> {
   const messageHash = ethers.utils.solidityKeccak256(
     ["uint256", "bytes", "uint8[]", "bool"],
-    [nonce, groupPubKey, inactiveMembersIndices, failedHeartbeat]
+    [nonce, groupPubKey, inactiveMembersIndices, operatorInactivity]
```

Comment:
Why are we changing it? The last component is a boolean flag whether heartbeat failed:
https://github.com/keep-network/keep-core/blob/8dc2df00236d2a09f038187c83fd8441aa731d63/solidity/ecdsa/contracts/libraries/EcdsaInactivity.sol#L40-L50

Reply:
[6ae3265](https://github.com/keep-network/keep-core/pull/2860/commits/6ae32656cd66c38c0d03beb3525bfe89e9fe5a03)